lib/publiccloud/azure.pm

```diff
@@ -95,7 +95,7 @@ sub find_img {
         my $definition = get_required_var('DISTRI') . '-' . get_required_var('FLAVOR') . '-' . get_required_var('VERSION');
         $definition = get_var("PUBLIC_CLOUD_AZURE_IMAGE_DEFINITION", uc($definition));
         $json = script_output("az sig image-version show --resource-group '$resource_group' --gallery-name '$gallery' " .
-              "--gallery-image-definition '$definition' --gallery-image-version '$version'", timeout => 60 * 30);
+              "--gallery-image-definition '$definition' --gallery-image-version '$version'", proceed_on_failure => 1, timeout => 60 * 30);
```

Comment:
what if something goes wrong and it doesnt return a json?

Reply:
ok, I see this is what you do below with: 
`record_info('IMGV NOT-FOUND', "Cannot find image-version $name. Need to upload it.\n$@") if ($@);`

Reply:
Yes :smile: